eq.js

```diff
@@ -14,16 +14,25 @@ const __ = require('./_internal/placeholder')
  * var value any,
  *   leftCompare any,
  *   rightCompare any,
- *   left (value=>Promise|leftCompare)|leftCompare,
- *   right (value=>Promise|rightCompare)|rightCompare
+ *   left value|(value=>Promise|leftCompare)|leftCompare,
```

Comment:
I'd like to amend this entire specscript section to read 

```
eq(leftValue any, rightValue any) -> boolean
eq(leftValue any, right function)(value any) -> Promise|boolean
eq(left function, rightValue any)(value any) -> Promise|boolean
eq(left function, right function)(value any) -> Promise|boolean
```

could we also make changes to follow a similar pattern for gt, gte, lt, and lte?